pkgs/tools/filesystems/9pfs/default.nix

```diff
@@ -10,6 +10,13 @@ stdenv.mkDerivation {
     sha256 = "007s2idsn6bspmfxv1qabj39ggkgvn6gwdbhczwn04lb4c6gh3xc";
   };
 
+  # Upstream development has stopped and is no longer accepting patches
+  # https://github.com/mischief/9pfs/pull/3
+  postPatch = lib.optionalString stdenv.isDarwin ''
+    sed -i '1s/^/typedef unsigned long ulong;\n/' libc.h
+    substituteInPlace 9pfs.c --replace 'MSIZE' 'MSIZE_9P'
```

Comment:
I would prefer a patch instead of a silently failing sed.

Reply:
Fixed.